### PR TITLE
Fix issues in formatting and README

### PR DESCRIPTION
**Pull request by riyafa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ service<http:Service> LocalShop bind LocalShopEP {
 Navigate to `pass-through` and run the following command in the command line to start `passthrough.bal`.
 
 ```bash
-   $ ballerina run passthrough.bal
+   $ ballerina run passthrough
```
Comment (MaryamZi) [inline README.md]: need to update the comment "Navigate to `pass-through` and run the following command in the command line to start `passthrough.bal`." too?

Comment (riyafa) [reply README.md]: Maybe not because this command also will start the services in `passthrough.bal` file

Comment (MaryamZi) [reply README.md]: Then shouldn't the comment be
"Navigate to `pass-through-messaging/guide` and run the following command in the command line to start  the services in `passthrough.bal`." ?

Comment (riyafa) [reply README.md]: Fixed

**Pull request merged**